### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'addCucumberPreprocessorPlugin')

### DISCUSSION THREAD

**Discussion by chithrabk** [cypress-io/cypress#24770] [CI setup]
Hi all,
I am trying to setup the framework using cypress+cucumber with typescript but Im getting the below error
![image](https://user-images.githubusercontent.com/65008300/203293940-372dd428-3383-47a6-a19d-fcfc6b161e1c.png)
Please help to resolve this...

Thanks in advance

**Comment by samixchoumi** [comment]
@chithrabk does your `package.json` have that ? `"type": "module"`,
